### PR TITLE
Add terminal change event with port info

### PR DESCRIPTION
**Pull request by prusse-martin**
Also allows to control port visibility.

### CODE REVIEW

File: qmxgraph/page/api.js [hunk 1]
```diff
@@ -597,6 +592,42 @@ graphs.Api.prototype.isVisible = function isVisible (cellId) {
...
+ * @param {string} portName Name of a port in the cell.
+ * @param {boolean} visible Visibility state.
+ * @throws {Error} Unable to find cell.
+ */
+graphs.Api.prototype.setPortVisible = function setPortVisible (cellId, portName, visible) {
+    "use strict";
+
+    var graph = this._graphEditor.graph;
+    var model = graph.getModel();
+    this._findCell(model, cellId);  // Cell missing detection.
```
[the remark this answers is not in the document]
Comment (prusse-martin) [reply qmxgraph/page/api.js]: "cell missing detection" == "detects that the cell does not exist and throw an error in that case"

**Comment by nicoddemus** [comment]
I would say "CHANGELOG?", but nevermind. 😞

**Comment by prusse-martin** [comment]
> 
> 
> I would say "CHANGELOG?", but nevermind. 😞

Our https://github.com/ESSS/qmxgraph/blob/master/HISTORY.rst is quite long...

**Comment by williamjamir** [comment]
> > I would say "CHANGELOG?", but nevermind. 😞
> 
> Our [/HISTORY.rst@`master`](https://github.com/ESSS/qmxgraph/blob/master/HISTORY.rst) [](/ESSS/qmxgraph/blob/HEAD@{2020-02-28T12:59:04Z}/HISTORY.rst) is quite long...


We didn't release a 0.2.0 on PyPI yet.
So I think that we could start to populate from now on.

Perhaps we could add an "Initial Release" for 0.2.0 and release a new version on PyPI.

```python
=======
History
=======

0.2.0 (2020-02-28)
-----------------

* Initial release

0.1.0 (2017-01-03)
------------------

* Internal release.
```

**Comment by nicoddemus** [comment]
I think this would make sense to do once we start to work with this module as package... otherwise we don't really have incentives to make new releases until then.

**Comment by nicoddemus** [comment]
@prusse-martin ping

**Comment by prusse-martin** [comment]
We will postpone this PR merge a bit.

**Pull request merged**